UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/appsettings.json

```diff
@@ -57,6 +57,7 @@
         "EncUpdateListFileMimeType": "text/csv",
         "IsoFileMimeType": "application/x-raw-disk-image",
         "Sha1FileMimeType": "text/plain",
-        "ZipFileMimeType": "application/zip"
+        "ZipFileMimeType": "application/zip",
+        "PosDVDVolumeIdentifier": "M01X0{0}"
```

Comment:
Should this be consistent with the other entry in fulfilment?

i.e. "M0{0}X02"